tools/icu/shrink-icu-src.py

```diff
@@ -48,16 +48,18 @@ def icu_ignore(dir, files):
         ign = ign + files
         # except...
         ign.remove('source')
-        ign.remove('license.html')
-        ign.remove('LICENSE')
+        if 'license.html' in ign:
+            ign.remove('license.html')
+        if 'LICENSE' in ign:
+            ign.remove('LICENSE')
```

Comment:
out of my head but perhaps `if any(item in ign for item in('LICENSE', 'license.html'))`